data/raw/keybindings.json

```diff
@@ -304,6 +304,13 @@
       { "input_method": "keyboard_code", "key": "d", "mod": [ "shift" ] }
     ]
   },
+  {
+    "type": "keybinding",
+    "id": "CHANGE_MAPEXTRA_CHARACTER",
+    "category": "AUTO_NOTES",
+    "name": "Change a symbol for map extra",
+    "bindings": [ { "input_method": "keyboard_any", "key": "=" } ]
```

Comment:
Giving this a dedicated key seems like quite a stretch, if anything I'd expect it to be sufficient to access it via Esc -> something -> something